tests/core/meson.build

```
@@ -68,9 +68,26 @@ if host_arch == 'arm64'
   endif
 endif
 
+gum_tests_core_link = []
+
+if host_os_family == 'linux'
+  if host_arch == 'x86' or host_arch == 'x86_64'
```

```suggestion
if host_os_family == 'linux' and host_arch in ['x86', 'x86_64']
```

(At least I think this kind of syntax is supported.)